.github/labels.yml

```diff
@@ -319,11 +319,3 @@
 - name: "❔question❔"
   description: "I have a proposal or question about things, but nothing is broken."
   color: "293028"
-
-- name: "prickle💢"
-  description: "This issue should be skipped by the auto-responder."
-  color: "FFFFFF"
-
-- name: "smolder🍲"
-  description: "Lower-priority and/or future idea"
-  color: "8B0000"
```

Comment:
These labels being removed is due to this file (`.github/labels.yml`) being manually edited, but this file is automatically generated (this is explained in the [header of this file](https://github.com/exercism/python/blob/main/.github/labels.yml#L1-L14)).

To not have these labels be removed, they should instead be added to  https://github.com/exercism/python/blob/main/.appends/.github/labels.yml.

Reply:
https://github.com/exercism/python/pull/2970 fixes this